src/vm/vmrunner.ts

```diff
@@ -460,6 +496,28 @@ export class VMProgramRunner extends JDClient {
         return this._watch[sourceId]
     }
 
+    // breakpoints
+    setBreakpoints(breaks: string[]) {
+        breaks.forEach(b => {
```

Comment:
should you clear previous breakpoints here too?

Reply:
OK, so you will always send me the full set of breakpoints from Blockly (no incremental shit here)?